### DISCUSSION TITLE
Separate UI tab to see reacts/mentions/updates for threads you're part of

### DISCUSSION THREAD

**Discussion by Stvad** [element-hq/element-meta#933] [Product Feedback]
#### What would you like to do?

I'd like to easily follow/see updates to my conversation contexts

#### Why would you like to do it?
I don't want to miss people engaging with my messages

#### How would you like to achieve it?
Slack's implementation is to have 2 separate UI tabs: one for mentions and reacts and one for updates in threads you're part of. 
<img width="298" alt="image" src="https://user-images.githubusercontent.com/81347/204865392-7c839a31-1b5e-489e-818a-ddc2eb455cda.png">

I think this works well for reacts/mentions (in case of matrix, this should probably include all different relations types - e.g. replies, etc)

I'm less happy with the Threads tab. It's good to have a way to see that there were updates in any of the threads you're part of, but IMO it'd work better when combine with approach from Discords's/Zulip's - they'd display threads you're part of in the sidebar & highlight the ones that have new messages:
<img width="236" alt="image" src="https://user-images.githubusercontent.com/81347/204867866-2294d903-1ca2-4a37-8d18-dec5fd9326e3.png">

**Comment by MurzNN** [comment]
In the current Element UI we have a separate "Threads" tab, which is great!

But still missing the "Mentions & Reactions" tab, which should be much more useful than "Threads"!

In addition to Slack, a great UI implementation of this feature we can check in the MS Teams interface, there it is named as the "Activity" tab with filters by the activity type.
